models.go

```diff
@@ -22,6 +22,7 @@ type LogFormat struct {
 	Message   string   `json:"message"`
 	LogLevel  LogLevel `json:"log_level"`
 	Data      Data     `json:"data"`
+	Error     error    `json:"-"`
```

Comment:
I wasn't totally sure what to do with this when it was being marshaled, so I just omitted it